src/lib/ngsi/ContextElement.cpp

```diff
@@ -111,6 +111,32 @@ std::string ContextElement::render(ConnectionInfo* ciP, RequestType requestType,
 
 /* ****************************************************************************
 *
+* ContextElement::toJson - 
+*/
+std::string ContextElement::toJson(ConnectionInfo* ciP, NotificationFormat notifyFormat)
```

Comment:
Yes, true. I added the ConnectionInfo only for symmetry.

If it is true what you say that most toJson don't accepr ConnectionInfo as a parameter then perhaps it was not a good idea.

However, if we truly want symmetry (which is a very good thing) and only a few toJson() needs ciP, then all toJson() must accept it ...

15/8   without/with => 35% of toJsons accept ciP as a parameter.

So, if we want symmetry, I'll leave ciP and if not, then I'll remove it.
Votation! :-)